lib/pf/role.pm

```diff
@@ -678,6 +659,10 @@ sub shouldAutoRegister {
             return $role;
         }
     }
+    if ($args->{'profile'}->{'_autoregister'} eq "enabled") { 
```

Comment:
Fully agree with that wish, but I'd say for now it has to be `isenabled($args->{'profile'}->{'_autoregister'})` so we stay consistent in our code